lib/hummingbird_survey/models/survey_page_form.rb

```diff
@@ -168,23 +168,38 @@ def answer_value(value, item)
       end
     end
 
+    def final_question_hash(item)
+      base_itemable_data(item).merge!({
+        "label" => item.survey_itemable.label,
+        "question_type" => item.survey_itemable.question_type.to_s,
+        "value" => answer_value(all_answer_data["question_#{item.survey_itemable.id}"], item),
+        "question_id" => item.survey_itemable.id,
+      })
+    end
+
+    def final_section_hash(item)
```

Comment:
@cgratigny : this is a little misleading. it _seems_ like it should result in the final section hash, with all the items, but it's really the base upon which the final section hash is built (starting down on line 204). Perhaps something like `base_section_hash` or `starting_section_hash` or something?

Reply:
@Onikoroshi got it.